src/Components/CategoryCard/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState} from 'react';
+import './style.component.css'
+import Dropdown from './dropdown'
+
+
+function CategoryCard({title, imageUrl}) {
+    const [active, setActive] = useState(false);
+
+
+    function toggle(){
```

Comment:
Pela minha experiencia de React (que não é muita) geralmente essas funções de onClick a gente coloca o nome de "handleAlgumaCoisa". Seria tu interessante usar o useCallback inclusive
```suggestion
    const handleClick = usecallback(() => {
```